src/lib/features/project-insights/project-insights-service.e2e.test.ts

```diff
@@ -0,0 +1,296 @@
+import dbInit, { type ITestDb } from '../../../test/e2e/helpers/database-init';
```

Comment:
this is only copy paste of DORA tests from the project service. Those tests will soon be gone from the project service @FredrikOseberg